### PR TITLE
CIV-15040 LipvLip - Update JBA to RTL through CJES Service

### PR DESCRIPTION
**Pull request by kalachandrasekar1**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-15040

### Change description ###
Once Applicant accepted or rejected the repayment plan and requested ccj, update the end state to All_Final_Order_Issued and send the active judgement to RTL through CJES service

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/ClaimantResponseCuiCallbackHandler.java [hunk 1]
```diff
@@ -126,6 +129,11 @@ private CallbackResponse aboutToSubmit(CallbackParams callbackParams) {
         UnavailabilityDatesUtils.rollUpUnavailabilityDatesForApplicant(
             builder, featureToggleService.isUpdateContactDetailsEnabled());
 
+        if (JudgmentAdmissionUtils.getLIPJudgmentAdmission(caseData) && featureToggleService.isJudgmentOnlineLive()) {
```
Comment (ShwetaTandel-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/ClaimantResponseCuiCallbackHandler.java:132]: A general comment - check if feature flag is live first, one less condition to check if its not on.

Comment (kalachandrasekar1) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/ClaimantResponseCuiCallbackHandler.java]: yes, it's fixed

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by ShwetaTandel-hmcts — DISMISSED** [review]

**Review by neeta-hmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowFlag.java [hunk 2]
```diff
@@ -20,6 +20,7 @@ public enum FlowFlag {
     LIP_CASE,
     CLAIM_ISSUE_HWF,
     LIP_JUDGMENT_ADMISSION,
+    JBA_ONLINE_LIVE_ENABLED,
```
Comment (Azam-Hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowFlag.java:23]: I think it shuld be named as "JO" then "JBA" ?

Comment (kalachandrasekar1) [reply src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowFlag.java]: Updated the name to JO_ONLINE_LIVE_ENABLED

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Pull request merged**